nuclear/b.py

```diff
@@ -72,9 +72,9 @@
 
 if __name__ == "__main__":
 
-    # Add our builtin tools to the path and user tools
-    sys.path.append(user_tools_path)
-    sys.path.append(nuclear_tools_path)
+    # Prepend nuclear and user tools to the path, so we prefer our packages
+    sys.path.insert(0, user_tools_path)
+    sys.path.insert(0, nuclear_tools_path)
```

Comment:
This is backward, we should prefer our own tools. But currently, this will prefer the built-in nuclear ones. For example this will mess up the `./b module generate` tool by not using the built in NUbots one